addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -119,6 +119,15 @@
                     </button>
                 </div>
             </div>
+            <div class="mt-4 text-info o_wslides_js_lesson_quiz_resourceinfo d-none">
+                <div t-if="widget.quiz.slideResources">
+                    <span t-if="widget.quiz.slideResources.length > 0">Need help? Review related content:</span> 
+                    <div t-foreach="widget.quiz.slideResources" t-as="resource">
+                        <div class="">-<a t-att-href="resource.link or resource.download_url"
+                                 t-esc="resource.text_value"/></div>
```

Comment:
Could you instead update css to reduce the spacing between elements while keeping `<ul>` and `<li>` ?
It should work by setting `list-style-type: "-";` on the tagged `ul`'s and updating margins for the `li` elements.